st_tools_bonus_activity.Rmd

```diff
@@ -0,0 +1,578 @@
+---
+title: "Bonus activity: Using the command-line for single-cell Spatial Transcriptomics analysis"
+output:
+  html_document:
+    toc: true
+    toc_float:
+      toc_collapsed: true
+    toc_depth: 3
+    number_sections: false
+css: style.css
+editor_options:
+  markdown:
+    wrap: 72
+---
+
+```{r echo = FALSE, show = FALSE}
+knitr::opts_chunk$set(number_sections = FALSE)
+```
+
+To continue to practice using `spatialGE` using R, you can access our PositCloud instance until April 27th. See the [clinical workshop activity guidance](https://hutchdatascience.org/UMich_ITN_Workshop/clinical_data_activity.html) for instructions on how to connect.
+
+Alternatively you can download and install R and RStudio to use on your local machine or see if your institution has a sever available with R and R Studio.
```

Comment:
Alternatively you can download and install R and RStudio to use on your local machine or see if your institution has a sever available with R and RStudio. See this [link for information on installations](https://jhudatascience.org/intro_to_r/modules/Setup/Setup.html). You could also consider using another cloud-based option like [AnVIL](https://jhudatascience.org/AnVIL_Book_Getting_Started/starting-rstudio.html) or [Galaxy](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/rstudio/tutorial.html).